auth/package.json

```diff
@@ -41,5 +41,8 @@
         "test:validate": "sam validate",
         "test:integration": "vitest run --project integration",
         "test:functional": "vitest run --project functional"
+    },
+    "dependencies": {
+        "aws-jwt-verify": "^5.1.0"
```

Comment:
Should this not be a dependency of the `shared-signal-authorizer` as opposed to the entire `auth` directory?